src/Nancy/Session/CookieBasedSessions.cs

```diff
@@ -95,7 +95,7 @@ public static IObjectSerializerSelector Enable(IPipelines pipelines, Cryptograph
         {
             var cookieBasedSessionsConfiguration = new CookieBasedSessionsConfiguration(cryptographyConfiguration)
             {
-                Serializer = new DefaultObjectSerializer()
+                Serializer = new DefaultObjectSerializer(new AppDomainAssemblyCatalog())

```

Comment:
Again, I'm not sure this should be hard coded. Also, what happens to caching here?